bin/helpers/possibleLocations.js

```diff
@@ -0,0 +1,39 @@
+const path = require('path');
+
+const PROCESS_ENV_HOME = process.env.HOME;
+const PROCESS_ENV_APPDATA = process.env.APPDATA;
+const PROCESS_ENV_XDG_CONFIG_HOME = process.env.XDG_CONFIG_HOME;
```

Comment:
I don't know where ths ` process.env.XDG_CONFIG_HOME` and `process.env.APPDATA;` point, because I [read](https://nodejs.org/api/os.html#os_os_homedir) we can use `os.homeDir()` to simplify the `home` path no matter SO.

Otherwise, we can extract that const in its own file.

Reply:
Sounds like a good idea, but we need to test this in different os matrix in the CI here
https://github.com/rajasegar/alacritty-themes/blob/11a5bf2008760db0c4cce8ba5f2980501ad40a8a/.github/workflows/main.yml#L14

Reply:
I don't know if windows is a free ’OS matrix`, I'll search more about it.